include/deal.II/base/function_parser.h

```diff
@@ -328,10 +328,17 @@ class FunctionParser : public AutoDerivativeFunction<dim>
 
   /**
    * The muParser objects for each thread (and one for each component). We are
-   * storing unique_ptr so that we don't need to include the definition of
+   * storing shared_ptr so that we don't need to include the definition of
```

Comment:
Ah, I missed that part. Let me just change this back -- I think documenting the most up-to-date state is the correct solution; we comment on the special case in the `//` comment below.